controller/search_blackbox_test.go

```diff
@@ -1226,6 +1226,7 @@ func (s *searchControllerTestSuite) TestIncludedParents() {
 					}
 					includedData := make([]app.WorkItem, len(result.Included))
 					for i, v := range result.Included {
+						i = len(result.Included) - 1 - i
```

Comment:
Please do `j := len(result.Included) - 1 - i`. It is cleaner.

Reply:
done in this commit aadae24